### PR TITLE
Fixing instructions

### PR DESCRIPTION
**Pull request by leahbannon**
Adjusting fix to instructions based on @benbalter's suggestion https://github.com/github/government.github.com/pull/183#issuecomment-28233038

### CODE REVIEW

File: community.md [hunk 1]
```diff
@@ -29,11 +29,19 @@ This website is [open source](https://github.com/github/government.github.com),
 
 * Log in to [GitHub](https://github.com){: target=blank}
 * Go to [this repository](https://github.com/github/government.github.com) and fork it (button to the top right of the page).
-* In your forked version, click on the `_config.yml` file.
+* In your forked version, click the branch dropdown and in the "Fnid or create a branch..." field, create a new branch (for example, "adding myagencyname").
+* Still in your forked version, click on the `_config.yml` file.
```
Comment (nschonni) [inline community.md]: Oops, this has moved now to [_data/organizations.yml](https://github.com/github/government.github.com/blob/gh-pages/_data/organizations.yml) :wink:

**Comment by leahbannon** [comment]
@nschonni I edited to '_data/organizations.yml' based on your note in my forked version - did that automatically update the pull request?

**Comment by nschonni** [comment]
Yup, and that also might answers your question on "why to use a branch". If you were working off your master, you wouldn't be able to change anything else without affecting any open PR to that branch

**Comment by leahbannon** [comment]
:+1:

**Comment by benbalter** [comment]
:metal: Thanks for this.

**Pull request merged**